SPECS/mariner-rpm-macros/macros.python-srpm

```diff
@@ -9,9 +9,8 @@
 # use the non-underscored macros to refer to Python in spec, etc.
 %python3 %__python3
 
-# For backwards compatibility only
-# See the comments in https://src.fedoraproject.org/rpms/python-rpm-macros/pull-request/22
-%__python /usr/bin/python
+# Set default python macro to the version of python shipped with the OS
+%__python /usr/bin/python3
 
 # Users can use %%python only if they redefined %%__python (e.g. to %%__python3)
```

Comment:
This comment is no longer correct. I think we should fix the definition of `%python` to be the following instead of the lua code currently there:

`%python %__python`

Reply:
That's a good call, I was only looking at %__python, but I agree %python should be set. Fixed